.github/workflows/docs.yml

```diff
@@ -69,6 +69,28 @@ jobs:
           path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
           key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
 
+      # Build bindings/nodejs docs
+      - name: Corepack
+        working-directory: bindings/nodejs
+        run: corepack enable
+
+      - name: Install dependencies
+        working-directory: bindings/nodejs
+        run: yarn install
+
+      - name: Build
+        working-directory: bindings/nodejs
+        run: yarn build
+
+      - name: Build bindings/nodejs Docs
+        working-directory: bindings/nodejs
+        run: yarn docs
+
+      - name: Copy docs into build
+        run: |
+          cp -r ./bindings/nodejs/docs ./website/static/docs/js
```

Comment:
Is `js` and `nodejs` the same? Maybe we should use `nodejs` here?

Reply:
Make sense.